apps/progs.pl

```diff
@@ -131,6 +131,7 @@
     "aes-128-cbc", "aes-128-ecb",
     "aes-192-cbc", "aes-192-ecb",
     "aes-256-cbc", "aes-256-ecb",
+    "aria",
```

Comment:
Errrr...  the `enc` command uses the EVP interface, and what `progs.pl` does here is to produce aliases for algorithms that can be found with `EVP_get_cipherbyname()`.